chef/cookbooks/network/recipes/default.rb

```diff
@@ -453,6 +500,22 @@ def kill_nic(nic)
         action :delete
       end
     end
-
+    if node[:platform] == "suse" && node[:platform_version].to_f < 12.0
```

Comment:
I'm fine with changing this, as long as we fully test an upgrade from before the patch to after the patch.